.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,25 @@
-Describe in one or two sentences why you're opening this issue:
+### Description:
+<!-- Describe in one or two sentences why you're opening this issue: -->
+> **REPLACE-ME**
```

Comment:
:smile: themselves? lol!
I placed it there for the formatting. But I now realize why that's wrong. So, I'll remove the blockquote formatting.